tests/e2e/unified_test.go

```diff
@@ -168,34 +168,36 @@ func TestAllInSeries(t *testing.T) {
 					events := h.Events
 
 					operationIDs := map[string]bool{}
-					for _, req := range events.HTTPEvents {
-						url := req.Request.URL
+					for _, event := range events.HTTPEvents {
 						id := ""
-						if index := strings.Index(url, "/v3/operations/"); index != -1 {
-							id = strings.TrimPrefix(url[index:], "/v3/operations/")
+						body := event.Response.ParseBody()
+						val, ok := body["name"]
+						if ok {
+							if strings.Contains(val.(string), "operations/") {
```

Comment:
Nit: Should be StartsWith if you're pairing with TrimPrefix, I think (or use Index if it doesn't have to be a prefix)